LSP-rust-analyzer.sublime-settings

```diff
@@ -1,14 +1,11 @@
 {
 	"settings": {
 		// Settings Not Related to rust-analyzer Server
-
 		// Whether or not to automatically close terminus panel/tab upon completion.
 		"rust-analyzer.terminusAutoClose": false,
 		// Whether or not to spawn a panel at the bottom, or a new tab
 		"rust-analyzer.terminusUsePanel": false,
-
 		// Rust-Analyzer Server Settings
-
```

Comment:
I like those blank lines. Otherwise the text merges with the description of the option which it is not.

Alternatively you could wrap those in `***` or something to make them stand out.